quickwit-indexing/src/actors/indexer.rs

```diff
@@ -168,9 +168,9 @@ impl IndexerState {
     ///
     /// This function will then create it, and can hence return an Error.
     async fn get_or_create_workbench<'a>(
-        &self,
+        &'a self,
```

Comment:
Wasn't it better before?

Reply:
I was getting random compile errors here when some other parts of the code were not also not compiling. @kstaken did too. I never got them after adding this lifetime constraint on `&self`.